tests/integration/multipass/conftest.py

```diff
@@ -97,3 +101,43 @@ def tmp_instance(
         capture_output=True,
         check=False,
     )
+
+
+@pytest.fixture(autouse=True, scope="module")
+def installed_multipass():
+    """Ensure multipass is installed, or skip the test if we cannot.
+
+    If the environment has CRAFT_PROVIDERS_TESTS_ENABLE_MULTIPASS_INSTALL=1,
+    force the installation of Multipass if uninstalled.
+    """
+    if os.environ.get("CRAFT_PROVIDERS_TESTS_ENABLE_MULTIPASS_INSTALL") == "1":
+        multipass.install()
```

Comment:
It's really scary to run tests and those tests use sudo and ask you for your password.

Even if one set an envvar (which may not fully understand).

Can't we go to the model of "using multipass if present, skipping if not, in this case telling the developer to check the README for installation instructions"?

Reply:
It defaults to that behavior (see below elif).

I cannot imagine anyone would honestly be surprised that it tried to install Multipass when setting CRAFT_PROVIDERS_TESTS_ENABLE_MULTIPASS_INSTALL and attempting to run the Multipass integration tests.

But I'll remove it, unless you can think of a foolproof envvar alternative :rofl:

Reply:
Ah! Here (and below) I misread/confused unit with integration tests, sorry!